ui/welcome/src/commonMain/kotlin/social/androiddev/welcome/composables/WelcomeScreen.kt

```diff
@@ -33,17 +33,40 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import social.androiddev.common.composables.buttons.MastodonButton
-import social.androiddev.common.composables.buttons.MastodonOutlinedButton
 import social.androiddev.common.theme.Blue
 import social.androiddev.common.theme.MastodonTheme
 import social.androiddev.common.utils.AsyncImage
 import social.androiddev.common.utils.loadImageIntoPainter
+import social.androiddev.welcome.navigation.WelcomeScreenComponent
 
 @Composable
 fun WelcomeScreen(
```

Comment:
A component doesn't not necessarily represent a complete screen, it may be a part of the screen. A better `Composable` naming pattern could be `XyzContent(component: XyzComponent)`.